docs-ref-services/key-vault.md

```diff
@@ -31,57 +30,51 @@ Use the management library to:
 ### Client library
 
 ```bash
-pip install azure-keyvault
+pip install  azure-keyvault-secrets azure-keyvault-keys azure-identity
 ```
 
 ## Examples
 
-The following examples use service principal authentication, which is the recommended sign in method
-for applications that connect to Azure. To learn about service principal authentication, see
-[Authenticate with the Azure SDK for Python](https://docs.microsoft.com/en-us/python/azure/python-sdk-azure-authenticate)
-
 Retrieve the public portion of an asymmetric key from a vault:
 
 ```python
-from azure.keyvault import KeyVaultClient
-from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.keys import KeyClient
 
-credentials = ServicePrincipalCredentials(
-    client_id = '...',
-    secret = '...',
-    tenant = '...'
-)
+credential = DefaultAzureCredential()
 
-client = KeyVaultClient(credentials)
+key_client = KeyClient("https://<vaultname>.vault.azure.net", credential)
 
-# VAULT_URL must be in the format 'https://<vaultname>.vault.azure.net'
-# KEY_VERSION is required, and can be obtained with the KeyVaultClient.get_key_versions(self, vault_url, key_name) API
-key_bundle = client.get_key(VAULT_URL, KEY_NAME, KEY_VERSION)
-key = key_bundle.key
+key = key_client.get_key("key-name")
```

Comment:
why is the key-name in ""? it is not the actual thing but a variable people need to substitute, no? shouldn't it have a comment like the version in the past:

```python
# KEY_NAME is a string that represents the name of your vault that you can find in the portal ""
key = key_client.get_key(KEY_NAME)
```
or something like that

Reply:
Key version is not required in latest SDK, comment is not necessary. Is there guidelines on how to represent input parameter? Looks like each author did it differently. I think string is better as it will compile as is

Reply:
I didn't mean the version as the KEY_VERSION but the approach that was described in the previous document :). I believe we have discussed this in person